.travis.yml

```diff
@@ -96,7 +98,7 @@ jobs:
       python: 3.7
       env:
         <<: *env_windows
-        JRNL_PYTHON_VERSION: 3.7.5
+        JRNL_PYTHON_VERSION: 3.7.7
```

Comment:
Can this PR update Python versions for all OSes instead of just Windows? Best to keep them all updated at once.

Reply:
Do you know where to find a list of the MasOS versions available? Windows versions are coming from [Chocolatey](https://chocolatey.org/packages/python/#versionhistory), but I'd assumed that the MacOS versions were tied to the base image...

Reply:
It looks like you can use Homebrew to install custom versions of Python, but that is beyond my experience... see http://www.pyslvs.com/blog/yuan_2018-09-22.html

The [Travis documentation](https://docs.travis-ci.com/user/reference/osx/) doesn't actually say what Python versions are available on what OSX images, just that at least one Python version will always be available.

Reply:
On the Mac builds, you can set `JRNL_PYTHON_VERSION` without having to modify anything else. Python comes pre-installed on macOS, so the build uses `pyenv install` to install whatever version the `JRNL_PYTHON_VERSION` is set to.